features/device-config/rest/api/src/main/java/org/opennms/features/deviceconfig/rest/api/DeviceConfigRestService.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
```

Comment:
I think we can combine rest api/impl maven modules into one. No need for both of them to exist

Reply:
Now, there is only one module.